roles/confluent.kafka_connect/tasks/connectors.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Ensure Plugin Dirs
+  file:
+    path: "{{item}}"
+    state: directory
+    group: "{{kafka_connect_group}}"
+    owner: "{{kafka_connect_user}}"
+    mode: '764'
+  when: item != '/usr/share/java'
+  with_items: "{{ kafka_connect.properties['plugin.path'].split(',') }}"
+  
+- name: Installing Local Plugins
```

Comment:
so the idea is customer moved a bunch of connectors onto each host manually before this playbook run? Would anyone really want to do that? I could see moving them to the ansible host and asking ansible to move them to the connect hosts